pull-kubernetes-integration is broken

https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-integration

example run from https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/61874/pull-kubernetes-integration/4390/ which just edits a md file.

```
I0329 10:24:38.400] Running tests for APIVersion: v1,admissionregistration.k8s.io/v1alpha1,admissionregistration.k8s.io/v1beta1,admission.k8s.io/v1beta1,apps/v1beta1,apps/v1beta2,apps/v1,authentication.k8s.io/v1,authentication.k8s.io/v1beta1,authorization.k8s.io/v1,authorization.k8s.io/v1beta1,autoscaling/v1,autoscaling/v2beta1,batch/v1,batch/v1beta1,batch/v2alpha1,certificates.k8s.io/v1beta1,extensions/v1beta1,events.k8s.io/v1beta1,imagepolicy.k8s.io/v1alpha1,networking.k8s.io/v1,policy/v1beta1,rbac.authorization.k8s.io/v1,rbac.authorization.k8s.io/v1beta1,rbac.authorization.k8s.io/v1alpha1,scheduling.k8s.io/v1alpha1,settings.k8s.io/v1alpha1,storage.k8s.io/v1beta1,storage.k8s.io/v1,storage.k8s.io/v1alpha1,
I0329 10:24:38.438] +++ [0329 10:24:38] Running tests without code coverage
W0329 10:24:38.590] k8s.io/kubernetes/vendor/gopkg.in/yaml.v2
W0329 10:24:38.591] k8s.io/kubernetes/vendor/github.com/spf13/pflag
W0329 10:24:39.036] k8s.io/kubernetes/vendor/github.com/spf13/cobra
W0329 10:24:39.385] k8s.io/kubernetes/vendor/github.com/ghodss/yaml
W0329 10:26:40.203] panic: test timed out after 2m0s
W0329 10:26:40.203] 
W0329 10:26:40.203] goroutine 33 [running]:
W0329 10:26:40.204] testing.startAlarm.func1()
W0329 10:26:40.204] 	/usr/local/go/src/testing/testing.go:1145 +0x139
W0329 10:26:40.204] created by time.goFunc
W0329 10:26:40.204] 	/usr/local/go/src/time/sleep.go:170 +0x52
W0329 10:26:40.204] 
W0329 10:26:40.204] goroutine 1 [chan receive]:
W0329 10:26:40.205] testing.(*T).Run(0xc4200d20f0, 0x6f9d3b, 0x11, 0x704670, 0xc42006fc01)
W0329 10:26:40.205] 	/usr/local/go/src/testing/testing.go:790 +0x59b
W0329 10:26:40.205] testing.runTests.func1(0xc4200d20f0)
W0329 10:26:40.205] 	/usr/local/go/src/testing/testing.go:1004 +0xa8
W0329 10:26:40.205] testing.tRunner(0xc4200d20f0, 0xc42006fd90)
W0329 10:26:40.206] 	/usr/local/go/src/testing/testing.go:746 +0x16d
W0329 10:26:40.206] testing.runTests(0xc4200b41a0, 0x8545c0, 0x13, 0x13, 0x857420)
W0329 10:26:40.206] 	/usr/local/go/src/testing/testing.go:1002 +0x522
W0329 10:26:40.206] testing.(*M).Run(0xc42006ff20, 0x14be428)
W0329 10:26:40.207] 	/usr/local/go/src/testing/testing.go:921 +0x207
W0329 10:26:40.207] main.main()
W0329 10:26:40.207] 	k8s.io/kubernetes/cmd/kubeadm/test/cmd/_test/_testmain.go:80 +0x1d4
W0329 10:26:40.207] 
W0329 10:26:40.207] goroutine 14 [syscall]:
W0329 10:26:40.208] syscall.Syscall6(0xf7, 0x1, 0x6659, 0xc4200719b8, 0x1000004, 0x0, 0x0, 0x0, 0x0, 0xc4200ce9e0)
W0329 10:26:40.208] 	/usr/local/go/src/syscall/asm_linux_amd64.s:44 +0x5
W0329 10:26:40.208] os.(*Process).blockUntilWaitable(0xc4200a0d50, 0x0, 0x0, 0x2)
W0329 10:26:40.208] 	/usr/local/go/src/os/wait_waitid.go:31 +0xcd
W0329 10:26:40.208] os.(*Process).wait(0xc4200a0d50, 0xc4200d4800, 0xc4200a0d50, 0xc42023d680)
W0329 10:26:40.209] 	/usr/local/go/src/os/exec_unix.go:22 +0x72
W0329 10:26:40.209] os.(*Process).Wait(0xc4200a0d50, 0xc420071bb0, 0xc4200d4790, 0xc420071ba8)
W0329 10:26:40.209] 	/usr/local/go/src/os/exec.go:115 +0x39
W0329 10:26:40.209] os/exec.(*Cmd).Wait(0xc4200d46e0, 0x0, 0x0)
W0329 10:26:40.210] 	/usr/local/go/src/os/exec/exec.go:446 +0xf6
W0329 10:26:40.210] os/exec.(*Cmd).Run(0xc4200d46e0, 0x2c, 0xc420098840)
W0329 10:26:40.210] 	/usr/local/go/src/os/exec/exec.go:289 +0x87
W0329 10:26:40.210] k8s.io/kubernetes/cmd/kubeadm/test/cmd.RunCmd(0x7ffe69375a70, 0x2c, 0xc420098840, 0x1, 0x1, 0xc4200987e0, 0x6a8840, 0xc4200b4720, 0x6e78e0, 0xc4200b4700, ...)
W0329 10:26:40.211] 	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kubeadm/test/cmd/util.go:37 +0x1e2
W0329 10:26:40.211] k8s.io/kubernetes/cmd/kubeadm/test/cmd.kubeadmReset(0x7ffe69375a70, 0x2c)
W0329 10:26:40.211] 	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kubeadm/test/cmd/init_test.go:23 +0xde
W0329 10:26:40.212] k8s.io/kubernetes/cmd/kubeadm/test/cmd.TestCmdJoinConfig(0xc42024e0f0)
W0329 10:26:40.212] 	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/cmd/kubeadm/test/cmd/join_test.go:46 +0xd8
W0329 10:26:40.212] testing.tRunner(0xc42024e0f0, 0x704670)
W0329 10:26:40.212] 	/usr/local/go/src/testing/testing.go:746 +0x16d
W0329 10:26:40.213] created by testing.(*T).Run
W0329 10:26:40.213] 	/usr/local/go/src/testing/testing.go:789 +0x569
W0329 10:26:40.213] 
W0329 10:26:40.213] goroutine 31 [IO wait]:
W0329 10:26:40.213] internal/poll.runtime_pollWait(0x7fb2c9e9ad30, 0x72, 0x83b1c0)
W0329 10:26:40.214] 	/usr/local/go/src/runtime/netpoll.go:173 +0x5e
W0329 10:26:40.214] internal/poll.(*pollDesc).wait(0xc4200aaa18, 0x72, 0x83b101, 0x8394c8, 0xffffffffffffffff)
W0329 10:26:40.214] 	/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0xfb
W0329 10:26:40.214] internal/poll.(*pollDesc).waitRead(0xc4200aaa18, 0xc4201dd201, 0x200, 0x200)
W0329 10:26:40.215] 	/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x4b
W0329 10:26:40.215] internal/poll.(*FD).Read(0xc4200aaa00, 0xc4201dd200, 0x200, 0x200, 0x0, 0x0, 0x0)
W0329 10:26:40.215] 	/usr/local/go/src/internal/poll/fd_unix.go:126 +0x23d
W0329 10:26:40.215] os.(*File).read(0xc4200a8130, 0xc4201dd200, 0x200, 0x200, 0x0, 0x0, 0xc4201c7548)
W0329 10:26:40.216] 	/usr/local/go/src/os/file_unix.go:216 +0x71
W0329 10:26:40.216] os.(*File).Read(0xc4200a8130, 0xc4201dd200, 0x200, 0x200, 0x0, 0x0, 0x1)
W0329 10:26:40.216] 	/usr/local/go/src/os/file.go:103 +0x8f
W0329 10:26:40.216] bytes.(*Buffer).ReadFrom(0xc4200df490, 0x83abc0, 0xc4200a8130, 0x7fb2c9e56028, 0xc4200df490, 0x1)
W0329 10:26:40.216] 	/usr/local/go/src/bytes/buffer.go:209 +0x1de
W0329 10:26:40.217] io.copyBuffer(0x83a780, 0xc4200df490, 0x83abc0, 0xc4200a8130, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
W0329 10:26:40.217] 	/usr/local/go/src/io/io.go:386 +0x349
W0329 10:26:40.217] io.Copy(0x83a780, 0xc4200df490, 0x83abc0, 0xc4200a8130, 0x0, 0x0, 0x0)
W0329 10:26:40.217] 	/usr/local/go/src/io/io.go:362 +0x7f
W0329 10:26:40.218] os/exec.(*Cmd).writerDescriptor.func1(0x488001, 0x0)
W0329 10:26:40.218] 	/usr/local/go/src/os/exec/exec.go:264 +0x69
W0329 10:26:40.218] os/exec.(*Cmd).Start.func1(0xc4200d46e0, 0xc4200b4780)
W0329 10:26:40.218] 	/usr/local/go/src/os/exec/exec.go:380 +0x35
W0329 10:26:40.218] created by os/exec.(*Cmd).Start
W0329 10:26:40.219] 	/usr/local/go/src/os/exec/exec.go:379 +0xa6c
W0329 10:26:40.219] 
W0329 10:26:40.219] goroutine 32 [IO wait]:
W0329 10:26:40.219] internal/poll.runtime_pollWait(0x7fb2c9e9adf0, 0x72, 0x83b1c0)
W0329 10:26:40.219] 	/usr/local/go/src/runtime/netpoll.go:173 +0x5e
W0329 10:26:40.220] internal/poll.(*pollDesc).wait(0xc4200aaab8, 0x72, 0x83b101, 0x8394c8, 0xffffffffffffffff)
W0329 10:26:40.220] 	/usr/local/go/src/internal/poll/fd_poll_runtime.go:85 +0xfb
W0329 10:26:40.220] internal/poll.(*pollDesc).waitRead(0xc4200aaab8, 0xc4201dd401, 0x200, 0x200)
W0329 10:26:40.220] 	/usr/local/go/src/internal/poll/fd_poll_runtime.go:90 +0x4b
W0329 10:26:40.221] internal/poll.(*FD).Read(0xc4200aaaa0, 0xc4201dd400, 0x200, 0x200, 0x0, 0x0, 0x0)
W0329 10:26:40.221] 	/usr/local/go/src/internal/poll/fd_unix.go:126 +0x23d
W0329 10:26:40.221] os.(*File).read(0xc4200a8148, 0xc4201dd400, 0x200, 0x200, 0x0, 0x0, 0xc4201c7d48)
W0329 10:26:40.221] 	/usr/local/go/src/os/file_unix.go:216 +0x71
W0329 10:26:40.222] os.(*File).Read(0xc4200a8148, 0xc4201dd400, 0x200, 0x200, 0x0, 0x0, 0x1)
W0329 10:26:40.222] 	/usr/local/go/src/os/file.go:103 +0x8f
W0329 10:26:40.222] bytes.(*Buffer).ReadFrom(0xc4200df500, 0x83abc0, 0xc4200a8148, 0x7fb2c9e56028, 0xc4200df500, 0x555501)
W0329 10:26:40.222] 	/usr/local/go/src/bytes/buffer.go:209 +0x1de
W0329 10:26:40.223] io.copyBuffer(0x83a780, 0xc4200df500, 0x83abc0, 0xc4200a8148, 0x0, 0x0, 0x0, 0x6f9c2b, 0x11, 0x0)
W0329 10:26:40.223] 	/usr/local/go/src/io/io.go:386 +0x349
W0329 10:26:40.223] io.Copy(0x83a780, 0xc4200df500, 0x83abc0, 0xc4200a8148, 0x0, 0x0, 0xc4201c7fc0)
W0329 10:26:40.223] 	/usr/local/go/src/io/io.go:362 +0x7f
W0329 10:26:40.224] os/exec.(*Cmd).writerDescriptor.func1(0x488001, 0x7df480)
W0329 10:26:40.224] 	/usr/local/go/src/os/exec/exec.go:264 +0x69
W0329 10:26:40.224] os/exec.(*Cmd).Start.func1(0xc4200d46e0, 0xc4200b47c0)
W0329 10:26:40.224] 	/usr/local/go/src/os/exec/exec.go:380 +0x35
W0329 10:26:40.224] created by os/exec.(*Cmd).Start
W0329 10:26:40.225] 	/usr/local/go/src/os/exec/exec.go:379 +0xa6c
W0329 10:26:40.239] make[1]: *** [test] Error 1
W0329 10:26:40.243] make: *** [test-cmd] Error 1
I0329 10:26:40.343] FAIL	k8s.io/kubernetes/cmd/kubeadm/test/cmd	120.044s

```

--- a/cluster/kubeadm.sh
+++ b/cluster/kubeadm.sh
@@ -19,7 +19,6 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=${KUBE_ROOT:-$(dirname "${BASH_SOURCE}")/..}
-source "${KUBE_ROOT}/cluster/kube-util.sh"
 source "${KUBE_ROOT}/cluster/clientbin.sh"
 
 # If KUBEADM_PATH isn't set, gather up the list of likely places and use ls

